src/components/AudioThumbnail/AudioThumbnail.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <!-- Should be wrapped by a fixed-width parent -->
```

Comment:
Hmm... I wonder if this could be integrated into the component via a prop or something like that, rather than requiring the consumer to implement a specific parent each time. Or is there some reason to keep the width outside of the context of this component?

Reply:
Since the thumbnail can be used in a number of places, I thought it best to leave the positioning and resizing of the component to where it is used instead of having to modify this component when the layout changes or a new usage is added.